spyder/app/mainwindow.py

```diff
@@ -2897,7 +2918,7 @@ def _check_updates_ready(self):
         latest_release = self.worker_updates.latest_release
         error_msg = self.worker_updates.error
 
-        url_r = 'https://github.com/spyder-ide/spyder/releases'
+        url_r = __project_url__ + '/releases'

```

Comment:
Format string?

Reply:
Sure CAM do, but any particular advantage in this context (one variable string, one fixed string) to offset the lesser code clarity stemming from loss of sequentiality and greater complexity?